api docs + deprecations (#203)
latchbio/latch: docs/source/tutorials/metamage.md

In this tutorial, we will use the Latch SDK to build **metamage**, a workflow for taxonomic classification, assembly, binning and annotation of short-read host-associated metagenomics datasets.

## Prerequisites

* Install the Latch SDK.
* Understand basic concepts of a workflow through our [Quickstart](../getting_started/quick_start.md) and [Authoring your Own Workflow](../getting_started/authoring_your_workflow.md).

## What You Will Learn

> Bioinformatics workflows often deal with pipelines with tens to hundreds of potential tasks, where each task performs an analysis given some input.

Reviewer: ```suggestion
Bioinformatics workflows often deal with pipelines with tens to hundreds of tasks, where each task processes a set of input values.
```

    ]
```

To upload the full workflow on the Latch Platform, use:

```console
latch register --remote metamage_latch
```

The workflow will show up in [console.latch.bio/workflows](https://console.latch.bio/workflows).

## Viewing the DAG for subworkflows

When you navigate to the **Graph** page, you will see a condensed version of the workflow with subworkflows:
![MetaMage Condensed DAG](../assets/metamage-dag.png)

Pressing "Enter" on your keyboard allows you to view an expanded version of the workflow with tasks in every subworkflow and navigate between the expanded and condensed mode.
![MetaMage Expanded DAG](../assets/expanded-metamage-dag.png)

---

## Key Takeaways

* For medium to complex workflows with many tasks, it is recommended that you group tasks into a workflow.

> * Each workflow can be reused and imported into a larger workflow.

Reviewer: ```suggestion
* Each workflow can be reused as a subworkflow of a new, larger workflow.
```
